### DISCUSSION TITLE
进入程序会卡在启动页，需要鼠标在页面上点击一下才能进入welcome页

### DISCUSSION THREAD

**Discussion by nanchao70** [lobehub/lobehub#2953] [General Question] [answered]
### 📦 部署环境

Docker

### 📌 软件版本

1.0.8

### 💻 系统环境

Windows

### 🌐 浏览器

Chrome, Edge

### 🐛 问题描述

谷歌、edge都试过是一样的情况，自行部署和官方最新版的演示经过测试也是一样的问题。

### 📷 复现步骤

访问域名https://chat-preview.lobehub.com/ 就会卡在启动提示，需要鼠标在页面上进行点击操作，才会进入welcome。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
@nanchao70 能否给出更加详细的使用环境或者录屏？我们在自己的win上测了下似乎没这个问题

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
或者你是不是装了什么插件？试下匿名模式

[a comment by an automated account is not shown]

**Comment by nanchao70** [comment, marked as answer]
> 或者你是不是装了什么插件？试下匿名模式

试了无痕模式以及删除插件后，确实可以直接进入了。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
@nanchao70 OK， 能否告知是什么插件？我们后续写个 Q&A

[a comment by an automated account is not shown]

**Reply by nanchao70** [reply]
测试后是秘塔搜索，插件地址：https://chromewebstore.google.com/detail/%E7%A7%98%E5%A1%94ai%E6%90%9C%E7%B4%A2/amjmldchkefejkpgebhgioimmfkakgpo

**Comment by nanchao70** [comment]
felo插件也会导致同样的问题：https://chromewebstore.google.com/detail/felo-%E4%BD%A0%E7%9A%84%E5%85%8D%E8%B4%B9ai%E6%90%9C%E7%B4%A2%E5%8A%A9%E6%89%8B/fbnbeocmafoobaeodhmcgnammdeaoglg
